deploy/loderunner/local-cosmos/3-loderunner-api.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: loderunner-api
+  namespace: loderunner
+  labels:
+    app.kubernetes.io/name: loderunner-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: loderunner-api
+  template:
+    metadata:
+      labels:
+        app: loderunner-api
+    spec:
+      containers:
+        - name: app
+          image: k3d-registry.localhost:5000/loderunner-api:local
```

Comment:
I'm having trouble pulling this image in codespaces. The error from the pod is:

`Failed to pull image "k3d-registry.localhost:5000/loderunner-api:local": rpc error: code = NotFound desc = failed to pull and unpack image "k3d-registry.localhost:5000/loderunner-api:local": failed to resolve reference "k3d-registry.localhost:5000/loderunner-api:local": k3d-registry.localhost:5000/loderunner-api:local: not found`